openstack/maia/alerts/maia-prom.alerts

```diff
@@ -0,0 +1,36 @@
+groups:
+- name: maia-prom.alerts
+  rules:
+  - alert: MaiaPromFederationDown
+    expr: |
+        up{job=~"prometheus-openstack|
+            prometheus-infra-collector|
+            prometheus-vmware|
+            cronus-reputation-statistics"} < 1
+    for: 15m
+    labels:
+      severity: warning
+      support_group: observability
+      service: '{{ $labels.job }}'
+      context: federation
+      meta: 'Maia-Prom {{ $labels.job }} federation is down'
+    annotations:
+      description: 'Maia-Prom {{ $labels.job }} federation is down'
+      summary: Some Maia-Prom federation is down
+
+  - alert: MaiaPromServicesDown
+    expr: |
+        up{job=~"prometheus-maia-oprom|
+            prometheus-maia-oprom-thanos-query|
+            prometheus-maia-oprom-thanos-store|
+            prometheus-maia-oprom-thanos-compactor"} < 1
```

Comment:
Same as above. I would at least split into Maia and Prometheus.

Multiple line `|` need to be escaped if i am not mistaken. You can however write them in one line.